src/app/static/src/tests/integration/surveyAndProgram.itest.ts

```diff
@@ -41,7 +45,7 @@ describe("Survey and programme actions", () => {
         const formData = new FormData();
         formData.append('file', file);
 
-        await actions.uploadProgram({commit} as any, formData);
+        await actions.uploadProgram({commit, state : {}} as any, formData);
```

Comment:
The weird thing is that this test and the one below are *also* now timing out, even though they are not making any extra requests. This makes me think there is something weird going on.